TestBenches/InputRouter_test.cpp

```diff
@@ -403,7 +403,7 @@ void prepareInputStreams( ifstream * pInputStreams
 int main(int argc, char * argv[])
 {
   // default values for test bench 
-  int cLinkId = 6; 
+  int cLinkId = 8; 
```

Comment:
Is this an intentional changes of Sarah's HLS InputRouter code?

Reply:
Yes, I want the default IR that is being synthesised to be the one that's part of the simplest IR-VMR chain (two IRs and one VMR). I've already changed the VMR default in a previous PR.